### PR TITLE
FIX: if documents are filled do nothing

### PR DESCRIPTION
**Pull request by CJ-Wright**

### CODE REVIEW

**Comment by CJ-Wright** [comment]
Failing on stuff fixed in https://github.com/NSLS-II/bluesky/pull/788

**Comment by CJ-Wright** [comment]
@danielballan @tacaswell @arkilic ready for review!

File: databroker/tests/test_broker.py [hunk 1]
```diff
@@ -64,7 +64,7 @@ def test_uid_list_multiple_headers(db, RE):
     RE.subscribe(db.insert)
     uids = RE(pchain(count([det]), count([det])))
     headers = db[uids]
-    assert uids == [h['start']['uid'] for h in headers]
+    assert uids == tuple([h['start']['uid'] for h in headers])
```
Comment (danielballan) [inline databroker/tests/test_broker.py]: See my comment on #291.

File: databroker/tests/test_broker.py [hunk 2]
```diff
@@ -592,7 +592,11 @@ def __call__(*args, **kwrags):
     assert not ev2['filled']['image']
     ev2_filled = db.fill_event(ev2, inplace=False)
     assert ev2_filled['filled']['image'] == ev2['data']['image']
-
+    ev_ret2 = db.fill_events([ev], h.descriptors, inplace=True)
+    ev_ret3 =  db.fill_events(ev_ret2, h.descriptors, inplace=True)
+    ev = next(ev_ret3)
+    assert ev['filled']['image'] == datum
+    assert ev['filled']['image']
```
Comment (danielballan) [inline databroker/tests/test_broker.py]: What is the purpose of this line?

**Comment by danielballan** [comment]
> Failing on stuff fixed in NSLS-II/bluesky#788

It looks like 2/3 of the builds passed. I don't think the one failure had anything to do with  NSLS-II/bluesky#788. Databroker tests run against bluesky master, and NSLS-II/bluesky#788 was merged last week.

File: databroker/tests/test_broker.py (hunk 2 above)
Comment (CJ-Wright) [reply databroker/tests/test_broker.py]: I guess that isn't needed now that we are not putting `True` in things.

Comment (danielballan) [reply databroker/tests/test_broker.py]: I'm just confused about 598 and 599. If 598 is true and `datum` is not false-y, then 599 we be true also.

**Pull request merged**